File: linguagem C/Problema_terreno.c
#include <stdio.h>

int main()
{
    double comprimento, largura, valorq, valorterreno, area;
    
    printf("largura do terreno: ");
    scanf("%lf", &largura);
    printf("comprimento do terreno: ");
    scanf("%lf", &comprimento);
    printf("valor do metro quadrado: ");
    scanf("%lf", &valorq);
    //leitura das variaveis
    
    area = comprimento * largura;
    valorterreno = area * valorq;
    //manipulando variaveis
    
    printf("area do terreno: %.2lf\n", area);
    printf("valor do terreno: %.2lf\n", valorterreno);
    //saida de valores

    return 0;
}

File: linguagem Python/Problema_terreno.py
largura : float; comprimento : float; metroquadrado : float; area : float; preco : float
#variaveis flutuantes

largura = float(input("digite a largura do terreno: "))
comprimento = float(input("digite o comprimento do terreno: "))
metroquadrado = float(input("digite o valor do metro quadrado: "))
#lendo os valores das variaveis

area = largura * comprimento
preco = area * metroquadrado
#atribuindo o valor das variaveis como resultado a multiplicação das anteriormente lidas

print("area: {:.2f}m²".format(area))
print("preco: {:.2f} reais".format(preco))
#retornar os resultados da area e preço